src/sentry/conf/server.py

```diff
@@ -378,6 +378,8 @@
 CELERY_DEFAULT_EXCHANGE_TYPE = "direct"
 CELERY_DEFAULT_ROUTING_KEY = "default"
 CELERY_CREATE_MISSING_QUEUES = True
+CELERY_REDIRECT_STDOUTS = False
```

Comment:
All of this will just turn the `[MainProcess/Worker-1] I did a thing` into `name=celery.worker.job event=I did a thing` so if we think it's worth not changing the human celery logger, we can put this in a conditional.